addons/stock_account/models/product.py

```diff
@@ -8,18 +8,60 @@
 from collections import defaultdict
 from datetime import datetime
 
+COST_METHOD = [
+    ('standard', 'Standard Price'),
+    ('fifo', 'First In First Out (FIFO)'),
+    ('average', 'Average Cost (AVCO)'),
+]
+
+VALUATION = [
+    ('manual_periodic', 'Manual'),
+    ('real_time', 'Automated'),
+]
+
 
 class ProductTemplate(models.Model):
     _name = 'product.template'
     _inherit = 'product.template'
 
-    cost_method = fields.Selection(related="categ_id.property_cost_method", readonly=True)
-    valuation = fields.Selection(related="categ_id.property_valuation", readonly=True)
+    cost_method = fields.Selection(
```

Comment:
I understand that the valuation is configured through the group `group_stock_accounting_automatic`, but the cost_method should be configurable either on the product or on the company, we shouldn't need a category to configure those fields IMHO